rebar.config

```diff
@@ -31,3 +31,25 @@
       - (\"diameter_dict_util\":\"format_error\"/\"1\")
       - (\"diameter_dict_util\":\"parse\"/\"2\"))",
          []}]}.
+
+{dialyzer_extra_apps,
+ [
+  diameter
+ ]}.
+
+{dialyzer_warnings,
+ [
+  unmatched_returns,
+  error_handling,
+  race_conditions
```

Comment:
`race_conditions` can be slow[1]. I have not found it to be useful and avoid it because of the warning from Kostis.

[1] http://erlang.org/pipermail/erlang-bugs/2014-July/004518.html